test/ts/unit/collateralizer/scenarios/unsuccessful_seizure.ts

```diff
@@ -0,0 +1,158 @@
+    expectedRepaymentValueSchedule: [
+        {
+            timestamp: moment()
+                .subtract(8, "days")
+                .unix(),
+            expectedRepaymentValue: Units.ether(0.5),
+        },
+    ],
+    termsContract: (collateralizedContract: string, attacker: string) => collateralizedContract,
+    beneficiary: (originalBeneficiary: string, other: string) => originalBeneficiary,
+    from: (beneficiary: string, other: string) => beneficiary,
+    debtAgreementExists: true,
+    debtAgreementCollateralized: true,
+    validTermsContract: true,
+    succeeds: false,
+};
+
+export const UNSUCCESSFUL_SEIZURE_SCENARIOS: SeizeCollateralScenario[] = [
+    {
+        description: "Debt agreement does not exist",
+        ...defaultArgs,
+        debtAgreementExists: false,
+        agreementId: web3.sha3("Arbitrary 32 byte id for unsuccessful seizure scenario #0"),
+    },
+    {
+        description: "Debt agreement does not point to collateralized terms contract",
+        ...defaultArgs,
+        termsContract: (collateralizedContract: string, attacker: string) => attacker,
+        validTermsContract: false,
+        agreementId: web3.sha3("Arbitrary 32 byte id for unsuccessful seizure scenario #1"),
+    },
+    {
+        description:
+            "Debt agreement points to collateralized terms contract but was never itself collateralized",
+        ...defaultArgs,
+        debtAgreementCollateralized: false,
+        agreementId: web3.sha3("Arbitrary 32 byte id for unsuccessful seizure scenario #2"),
+    },
+    {
+        description: "(Grace Period = 0 Days) Debt is not currently in state of default",
+        ...defaultArgs,
+        valueRepaidToDate: _.last(defaultArgs.expectedRepaymentValueSchedule)
+            .expectedRepaymentValue,
+        gracePeriodInDays: new BigNumber(0),
+        agreementId: web3.sha3("Arbitrary 32 byte id for unsuccessful seizure scenario #3"),
+    },
+    {
+        description: "(Grace Period = 7 Days) Debt is not currently in state of default",
+        ...defaultArgs,
+        valueRepaidToDate: _.last(defaultArgs.expectedRepaymentValueSchedule)
+            .expectedRepaymentValue,
+        agreementId: web3.sha3("Arbitrary 32 byte id for unsuccessful seizure scenario #4"),
+    },
+    {
+        description:
+            "(Grace Period = 7 Days) Debt is currently in default, but grace period has not elapsed",
+        ...defaultArgs,
+        expectedRepaymentValueSchedule: [
+            {
+                timestamp: moment()
+                    .subtract(6, "days")
+                    .unix(),
+                expectedRepaymentValue: Units.ether(0.5),
+            },
+        ],
+        agreementId: web3.sha3("Arbitrary 32 byte id for unsuccessful seizure scenario #5"),
+    },
+    {
+        description: "(Grace Period = 90 Days) Debt is not currently in state of default",
+        ...defaultArgs,
+        gracePeriodInDays: new BigNumber(90),
```

Comment:
Again, probably my mistake, but this needs to specify `valueRepaidToDate` as being not in a state of default.  The default args specify the value repaid as being insufficient.

Reply:
It doesn't look like this is fixed?

Reply:
whoops, I had been trying to access the expected repayment value directly from defaultArgs, but hadn't committed it properly. Updated to `Units.ether(0.5)` - is that correct?